src/platform/stack/spec.rs

```diff
@@ -160,11 +172,18 @@ impl StackSpecV2 {
                     debug!("Installing manifest from Helm chart {}", helm_file);
 
                     // Read Helm chart YAML and apply templating
-                    let helm_chart = read_yaml_data_with_templating(helm_file, parameters)
-                        .await
-                        .context(TemplatedReadSnafu)?;
                     let helm_chart: HelmChart =
-                        serde_yaml::from_str(&helm_chart).context(YamlSnafu)?;
+                        match helm_file.into_path_or_url().context(PathOrUrlParseSnafu)? {
```

Comment:
I was actually thinking about combining local file AND remote file handling into the same `Client`. However, I wasn't sure if it is the right call. But I can include the local file handling and the `Client` then consumes a `PathOrUrl` instead of just `Url`.

Reply:
The client now takes a parameter of type `PathOrUrl` since 79da379